docker/config.yaml.mako

```diff
@@ -121,6 +121,13 @@ vars:
       # Specify any additional URL parameters that the print shall use for WMS calls
       wms_url_params:
         TRANSPARENT: 'true'
+      # Add a list of parameters which need to be kept in the requested URL used for the
```

Comment:
I think we should state explicitly that it's either wms_url_params or wms_url_keep_params

Suggestion:
If you want the print to keep some custom URL parameters directly from the reference_wms you have defined, then use the configuration option wms_url_keep_params instead of the configuration option wms_url_params.
In wms_url_keep_params, you can list which URL parameter values should be read from the reference_wms and used by the print.

Reply:
I changed it so that it now uses mostly yur wording except that even if the `wms_url_keep_params` is defined `wms_url_params` should still be kept for other base maps.